generators/app/templates/resource/tasks/config/webpack.js

```diff
@@ -48,7 +48,6 @@ module.exports = function(grunt) {
           disable: false,
           allChunks: true
         }),
-        new webpack.optimize.UglifyJsPlugin()
```

Comment:
I don't want to lose any functionality or regress here. So let's add the same functionality back in so we dont have a regression. See https://webpack.js.org/configuration/optimization/